### PR TITLE
Translation of zh-tw - part of Dogepedia

### PR DESCRIPTION
**Pull request by ukewea**
Fix Dogepedia article links in the following files:
- content/zh-tw/dogepedia.md
- content/zh-tw/dogepedia/_index.md

Translate following files:
- content/zh-tw/dogepedia/faq/dogecoin-and-coin-burning.md
- content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md
- content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md

Please report any issue if found, thanks!

### CODE REVIEW

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 [ author ]
   name = "Dogecoin"
 +++
 
-Oh, yes it does - and it always has!  
+噢，當然有啊，而且一直都有！
 
-Dogecoin’s primary purpose has always been to be used as a currency. Dogecoin was originally a fork of a cryptocurrency called Luckycoin - itself a fork of Litecoin. Billy Markus, co-creator of Dogecoin with Jackson Palmer, acted as a solo developer for the first Dogecoin Core releases. In 2014, Billy and Jackson left Dogecoin development, and a new development team was formed.  Over the years, the Dogecoin development team has consisted of a small group of maintainers coordinating the work of more than 40 contributors.
+狗狗幣的主要用途一直是被當作貨幣使用。狗狗幣最初是一種名為 Luckycoin 的加密貨幣（自身就是萊特幣的分叉）的分叉。Dogecoin Core 在一開始就只有 Billy Markus（狗狗幣的共同創造者）與 Jackson Palmer 2 位開發人員。2014 年，Billy 和 Jackson 離開了狗狗幣開發工作，接著一個新的開發團隊建立了起來。多年來，狗狗幣開發團隊由一小群維護人員組成，負責協調 40 多位貢獻者的工作。
```
Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:12]: `Dogecoin Core 在一開始就只有 Billy Markus 與 Jackson Palmer 2 位開發人員`
原文我讀起來感覺是 最初釋出第一版時還只有一人在做開發工作，然後查了一下，看到[這篇](https://dogecoin.com/dogepedia/articles/history-of-dogecoin/) 說：Jackson created the Dogecoin.com website and would have been, for a while, the public face of Dogecoin, while Billy worked as a solo developer on the first 4 releases.
因此在想如果這裡改為 `在第一版 Dogecoin Core 釋出時，開發人員只有 Billy Markus （他和 Jackson Palmer 同為狗狗幣創辦人）` 是否更準確~

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+在 2015 年，從 Dogecoin Core 1.10.0 開始，狗狗幣開發團隊決定將狗狗幣從比特幣上重新分叉出來，在比特幣的代碼庫之上重新套用萊特幣的變更。因此，縱觀其歷史，狗狗幣繼承了比特幣功能早已完整的代碼庫，並與其大部分的特性相同。它的獨特性來自更改過的參數、優化以及未從萊特幣或比特幣繼承的部分原始碼。這些變更始終著重在實用性和易用性，以便狗狗幣成為能用來進行日常消費的貨幣。
 
-In the future, the plan is to focus more efforts and resources to enhance Dogecoin so that it fulfills its aim to become a currency of the People. This will require a combination of part-time contributors, volunteers, but also full-time staff including developers and engineers.   
+未來預計會集中更多的精力和資源來強化狗狗幣，使其做到成為人民的貨幣的目標。要達成此目標需要集合包括非全職貢獻者、志願者以及包括開發人員和工程師在內的全職員工等人。
 
-The Dogecoin Foundation’s [Trailmap](https://foundation.dogecoin.com/trailmap/) lists various projects as the ecosystem works towards its shared aim of becoming a currency for the People, by the People! This Trailmap is not authoritative, but an indication of where to go that is open for community feedback.
+狗狗幣基金會的[路線圖](https://foundation.dogecoin.com/trailmap/)列出了各種專案，因為這個生態系正在努力實現其共同目標，即由「人民」使其成為人民的貨幣！此路線圖並不權威，只是指明了前進的方向，而且也歡迎社區的反饋。
```
Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:18]: There is zh-tw translated Trailmap🙂
https://foundation.dogecoin.com/zh-tw/trailmap/

Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:18]: I think that `社群` is more commonly used than `社區` between zh-tw users. Couldn't find a source on this though.😆 What do you think?

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 [ author ]
   name = "Dogecoin"
 +++
 
-Oh yes it does, and it always has! The simple fact is that *money has utility* - and Dogecoin *is* money! Indeed, Dogecoin is one of the few crypto**currencies** that has been used for this main purpose from day one.  
+噢，當然有啊，而且一直都有！一個簡單的事實是：*錢是有用處的* - 而狗狗幣*就是*錢！事實上狗狗幣是為數不多、從一開始就爲了這一目標而生的加密**貨幣**之一。
 
-Dogecoin was first meant to be a joke - and it served, and will serve, this purpose very well. It almost immediately became a means to tip and encourage others to go and achieve something: the Jamaican bobsled team, the water wells in Kenya, the Dogecoin car, and many more crazy initiatives the community supported over the years. As a tipping currency, thanks to its low fees and speed of transactions in comparison to competing chains for a long part of its existence, Dogecoin has never stopped working for its intended use. 
+狗狗幣在起初只是個玩笑 - 它在過去很好地遵循了這個傳統，在未來也會繼續。它幾乎是從一開始就被當作一種給小費的途徑，以及鼓勵人們做一些有意義的事情：贊助牙買加雪橇隊參加冬奧會、建造肯亞水井、建造狗狗幣汽車以及許多狗狗幣社區支持的瘋狂提案。狗狗幣始終保持了它誕生之初的用途 - 作為一種打賞貨幣，因爲相較於同期的其他貨幣，狗狗幣區塊鏈的交易費用低廉，且交易速度快速。
```
Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md:12]: 我想 `the Dogecoin car` 指的是那輛有名的[狗狗賽車](https://www.bnext.com.tw/article/62762/dogecoin-founder?)，你覺得要不要寫出是狗狗幣賽車呢?~

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+經過多年發展，在網路商店使用狗狗幣進行消費已經成為可能。隨著第三方廠商比如 BitPay、Coinbase 整合了狗狗幣支付、部分預付卡供應商支援狗狗幣做爲資金來源，狗狗幣的使用在過去 2 年呈現了爆炸性成長。商店還可以通過 JSON-RPC 介面直接整合到狗狗幣區塊鏈上。狗狗幣基金會正在與開發者們計劃透過建立一些流程化、標準化導入方式的函式庫，來使得前述做法更加便捷。
 
-Development is ongoing to enhance Dogecoin and make it fulfill its original promises while being competitive with other cryptos. With several groups of developers currently building for Dogecoin and its ecosystem, Dogecoin will find new purposes and uses.
+我們正在進行開發，以增強狗狗幣相較其他加密貨幣在實用性上的優勢。隨著好幾個團隊持續建設狗狗幣及其生態，狗狗幣還將找到新的實用場景。
 
-As per the [Dogecoin manifesto](https://foundation.dogecoin.com/manifesto/), Dogecoin focus lies on being useful, personable, welcoming, reliable, while maintaining a unique sense of fun.
+根據[狗狗幣宣言](https://foundation.dogecoin.com/zh-tw/manifesto/)，狗狗幣專注於實用、風度翩翩、好客以及穩健，同時要保持其獨特的趣味感。
```
Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md:18]: 個人覺得在這個地方，`熱情友好`或是`開放` 好像比`好客`容易融入語境/理解，你覺得呢?

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md (hunk 2 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:18]: OK, I'll update the URL, and change the translation of 'trailmap' from '路線圖' to '發展路線圖'

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md (hunk 1 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:12]: You are right, I misinterpret the sentence😭...

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md (hunk 2 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-developers.md:18]: Sure, I'll change translation of 'community' to '社群'

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md (hunk 4 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md:18]: The meaning of 'welcoming': friendly or making you feel welcome.
I'll change the translation of the word to '熱情友好'.

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md (hunk 3 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md:12]: OK, I'll change it.

**Comment by ukewea** [comment]
Thanks for the corrections @memetaverse!
I've update the .md files mentioned, please take a look.

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md (hunk 4 above)
Comment (memetaverse) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md]: '熱情友好' sounds great!

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
 
 [ author ]
   name = "Dogecoin"
 +++
 
-Oh yes it does, and it always has! The simple fact is that *money has utility* - and Dogecoin *is* money! Indeed, Dogecoin is one of the few crypto**currencies** that has been used for this main purpose from day one.  
+噢，當然有啊，而且一直都有！一個簡單的事實是：*錢是有用處的* - 而狗狗幣*就是*錢！事實上狗狗幣是為數不多、從一開始就爲了這一目標而生的加密**貨幣**之一。
 
-Dogecoin was first meant to be a joke - and it served, and will serve, this purpose very well. It almost immediately became a means to tip and encourage others to go and achieve something: the Jamaican bobsled team, the water wells in Kenya, the Dogecoin car, and many more crazy initiatives the community supported over the years. As a tipping currency, thanks to its low fees and speed of transactions in comparison to competing chains for a long part of its existence, Dogecoin has never stopped working for its intended use. 
+狗狗幣在起初只是個玩笑 - 它在過去很好地遵循了這個傳統，在未來也會繼續。它幾乎是從一開始就被當作一種給小費的途徑，以及鼓勵人們做一些有意義的事情：贊助牙買加雪橇隊參加冬奧會、建造肯亞水井、建造狗狗賽車以及許多狗狗幣社區支持的瘋狂提案。狗狗幣始終保持了它誕生之初的用途 - 作為一種打賞貨幣，因爲相較於同期的其他貨幣，狗狗幣區塊鏈的交易費用低廉，且交易速度快速。
```
Comment (memetaverse) [inline content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md:12]: We just need to change '社區' in this sentence to '社群' then you can merge! 😄

**Review by memetaverse — CHANGES_REQUESTED** [review]
Thanks for the great work!💛 I added the last comment, please take a look. And we're good to go.

File: content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md (hunk 5 above)
Comment (ukewea) [reply content/zh-tw/dogepedia/faq/dogecoin-has-no-utility.md]: OK, I've changed the word, please review for the last time (I hope)🙌.

**Review by memetaverse — APPROVED** [review]
Thank you! I've checked them again. All good, please merge! 😄🎉

**Pull request closed without merging**